modules/apps/commerce/commerce-service/service.xml

```diff
@@ -964,5 +964,6 @@
 		<exception>GuestCartItemMaxAllowed</exception>
 		<exception>GuestCartMaxAllowed</exception>
 		<exception>ProductBundle</exception>
+		<exception>RequiredCommerceChannelRel</exception>
```

Comment:
Sourceformat please. Wrong order.

Reply:
I ran it. The order appears to be correct